packages/create-react-native-library/src/index.ts

```diff
@@ -45,6 +51,12 @@ async function create(_argv: yargs.Arguments<Args>) {
 
   // Prefetch bob version in background while asking questions
   const resolveBobVersion = resolveBobVersionWithFallback(FALLBACK_BOB_VERSION);
+  const resolveNitroModulesVersion = resolveNitroModulesVersionWithFallback(
+    FALLBACK_NITRO_MODULES_VERSION
+  );
+  const resolveNitroCodegenVersion = resolveNitroCodegenVersionWithFallback(
+    FALLBACK_NITRO_CODEGEN_VERSION
+  );
```

Comment:
Seems unnecessary to create separate function for each thing and duplicating the logic. Instead make one function `resolveNpmPackageVersionWithFallback` and use as `resolveNpmPackageVersionWithFallback(packageName, fallbackVersion)`